code/mob/living/critter/small_animal.dm

```diff
@@ -2889,10 +2889,9 @@ var/list/mob_bird_species = list("smallowl" = /mob/living/critter/small_animal/b
 		var/mob/living/M = holder.owner
 		if (!holder)
 			return 1
-		M.visible_message("<span class='alert'><B>[M] does a funny little jiggle with their body and then vanishes into thin air!</B></span>") // MY ASCENSION BEGINS
+		M.visible_message("<span class='alert'><B>[M] does a funny little jiggle with their body and then disappears into thin air!</B?</span>") // MY ASCENCION BEGINS
```

Comment:
Dirty master :)))

Reply:
dammit, the thing didn't work! Both files should be relatively untouched _now_, though, sincere apologies.